src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3353,7 +3353,7 @@ namespace InternalHeatGains {
 
 				//For predefined tabular reports related to outside air ventilation
 				ZonePreDefRep( NZ ).isOccupied = true; //set flag to occupied to be used in tabular reporting for ventilation
-				ZonePreDefRep( NZ ).NumOccAccum += NumberOccupants *  Zone( NZ ).Multiplier * Zone( NZ ).ListMultiplier * TimeStepZone;
+				ZonePreDefRep( NZ ).NumOccAccum += NumberOccupants * TimeStepZone;
```

Comment:
Ah, OK.  And you've confirmed that the ACH values stay the same when multiplied?